### ISSUE TITLE
Broken column sizes for `neuro ps`

### ISSUE THREAD

**Issue by ayushkovskiy** [neuromation/platform-client-python#1032]
This output looks confusing:
```
ID                                        NAME  STATUS  WHEN         IMAGE            OWNER  DESCRIPTION  COMMAND
job-a96c65dc-8849-4d9b-81b4-93b65d72a77a      pending  a minute ago  image://neuromation/fastai:latest  <you>
```
columns size are broken

**Comment by ayushkovskiy** [comment]
together with the entry `<you>` looks very confusing

**Comment by shagren** [comment]
How to reproduce:
- kill all jobs
- run job without name and with large image name (35+ chars)

**Comment by asvetlov** [comment]
Changing just a column size doesn't help well.
Recently @dalazx suggested using multi-line rows if a cell cannot fit into defined size.
I think this is a valuable idea, we will have arbitrary long lines in the table anyway: image, command and job name are pretty huge often.
On the other hand, we don't need to display the *full* image name, there is `neuro status` command for it.
What we should show is reasonable small info that helps to distinguish one started job from others.
E.g. if the full `image://neuromation/fastai:latest` doesn't fit `.../fastai:...` can be enough for very many cases.

**Comment by shagren** [comment]
I played with `textable`. It can wrap cells if you have limited table width. 
Result can be(wide and normal terminal width):
1. Just with headers.

```
                   ID                           NAME          STATUS      WHEN                    IMAGE                    OWNER   DESCRIPTION   COMMAND 
job-2a43301c-aaf1-4249-b2fc-86139200bc66                     succeeded   Aug 16   alpine:latest                            <you>                 sleep 2m
job-45ff7ad4-ca48-41f8-8330-e22cb6288dad                     succeeded   Aug 28   image://neuromation/fastai:latest        <you>                         
job-9e76a6cc-7550-4c81-8d3f-89dc6b59b6be                     succeeded   Aug 28   image://neuromation/fastai:latest        <you>                         
job-8d49bc3c-b4f8-4067-89ec-fff074e2ee33   altis-processor   succeeded   Aug 30   image://shagren/altis-processor:latest   <you>                         
job-6a150ea0-969e-4f7a-b04d-e46cce9b593c   altis-processor   succeeded   Aug 30   image://shagren/altis-processor:latest   <you>                         
```
```
    ID          NAME       STATUS      WHEN      IMAGE      OWNER   DESCRIPTIO   COMMAND 
                                                                        N                
job-2a4330                succeeded   Aug 16   alpine:lat   <you>                sleep 2m
1c-aaf1-42                                     est                                       
49-b2fc-86                                                                               
139200bc66                                                                               
job-45ff7a                succeeded   Aug 28   image://ne   <you>                        
d4-ca48-41                                     uromation/                                
f8-8330-e2                                     fastai:lat                                
2cb6288dad                                     est                                       
job-9e76a6                succeeded   Aug 28   image://ne   <you>                        
cc-7550-4c                                     uromation/                                
81-8d3f-89                                     fastai:lat                                
dc6b59b6be                                     est                                       
job-8d49bc   altis-       succeeded   Aug 30   image://sh   <you>                        
3c-b4f8-40   processor                         agren/alti                                
67-89ec-ff                                     s-processo                                
f074e2ee33                                     r:latest                                  
job-6a150e   altis-       succeeded   Aug 30   image://sh   <you>                        
a0-969e-4f   processor                         agren/alti                                
7a-b04d-e4                                     s-processo                                
6cce9b593c                                     r:latest                                  
   
```

2. With header line
```
                   ID                           NAME          STATUS      WHEN                    IMAGE                    OWNER   DESCRIPTION   COMMAND 
=========================================================================================================================================================
job-2a43301c-aaf1-4249-b2fc-86139200bc66                     succeeded   Aug 16   alpine:latest                            <you>                 sleep 2m
job-45ff7ad4-ca48-41f8-8330-e22cb6288dad                     succeeded   Aug 28   image://neuromation/fastai:latest        <you>                         
job-9e76a6cc-7550-4c81-8d3f-89dc6b59b6be                     succeeded   Aug 28   image://neuromation/fastai:latest        <you>                         
job-8d49bc3c-b4f8-4067-89ec-fff074e2ee33   altis-processor   succeeded   Aug 30   image://shagren/altis-processor:latest   <you>                         
job-6a150ea0-969e-4f7a-b04d-e46cce9b593c   altis-processor   succeeded   Aug 30   image://shagren/altis-processor:latest   <you>                         
```
```
    ID          NAME       STATUS      WHEN      IMAGE      OWNER   DESCRIPTIO   COMMAND 
                                                                        N                
=========================================================================================
job-2a4330                succeeded   Aug 16   alpine:lat   <you>                sleep 2m
1c-aaf1-42                                     est                                       
49-b2fc-86                                                                               
139200bc66                                                                               
job-45ff7a                succeeded   Aug 28   image://ne   <you>                        
d4-ca48-41                                     uromation/                                
f8-8330-e2                                     fastai:lat                                
2cb6288dad                                     est                                       
job-9e76a6                succeeded   Aug 28   image://ne   <you>                        
cc-7550-4c                                     uromation/                                
81-8d3f-89                                     fastai:lat                                
dc6b59b6be                                     est                                       
job-8d49bc   altis-       succeeded   Aug 30   image://sh   <you>                        
3c-b4f8-40   processor                         agren/alti                                
67-89ec-ff                                     s-processo                                
f074e2ee33                                     r:latest                                  
job-6a150e   altis-       succeeded   Aug 30   image://sh   <you>                        
a0-969e-4f   processor                         agren/alti                                
7a-b04d-e4                                     s-processo                                
6cce9b593c                                     r:latest                                  
```

3. Header and vertical lines
```
                  ID                    |      NAME       |  STATUS   |  WHEN  |                 IMAGE                  | OWNER | DESCRIPTION | COMMAND 
=========================================+=================+===========+========+========================================+=======+=============+=========
job-2a43301c-aaf1-4249-b2fc-86139200bc66 |                 | succeeded | Aug 16 | alpine:latest                          | <you> |             | sleep 2m
job-45ff7ad4-ca48-41f8-8330-e22cb6288dad |                 | succeeded | Aug 28 | image://neuromation/fastai:latest      | <you> |             |         
job-9e76a6cc-7550-4c81-8d3f-89dc6b59b6be |                 | succeeded | Aug 28 | image://neuromation/fastai:latest      | <you> |             |         
job-8d49bc3c-b4f8-4067-89ec-fff074e2ee33 | altis-processor | succeeded | Aug 30 | image://shagren/altis-processor:latest | <you> |             |         
job-6a150ea0-969e-4f7a-b04d-e46cce9b593c | altis-processor | succeeded | Aug 30 | image://shagren/altis-processor:latest | <you> |             |         

```
```
    ID     |    NAME    |  STATUS   |  WHEN  |   IMAGE    | OWNER | DESCRIPTIO | COMMAND 
           |            |           |        |            |       |     N      |         
===========+============+===========+========+============+=======+============+=========
job-2a4330 |            | succeeded | Aug 16 | alpine:lat | <you> |            | sleep 2m
1c-aaf1-42 |            |           |        | est        |       |            |         
49-b2fc-86 |            |           |        |            |       |            |         
139200bc66 |            |           |        |            |       |            |         
job-45ff7a |            | succeeded | Aug 28 | image://ne | <you> |            |         
d4-ca48-41 |            |           |        | uromation/ |       |            |         
f8-8330-e2 |            |           |        | fastai:lat |       |            |         
2cb6288dad |            |           |        | est        |       |            |         
job-9e76a6 |            | succeeded | Aug 28 | image://ne | <you> |            |         
cc-7550-4c |            |           |        | uromation/ |       |            |         
81-8d3f-89 |            |           |        | fastai:lat |       |            |         
dc6b59b6be |            |           |        | est        |       |            |         
job-8d49bc | altis-     | succeeded | Aug 30 | image://sh | <you> |            |         
3c-b4f8-40 | processor  |           |        | agren/alti |       |            |         
67-89ec-ff |            |           |        | s-processo |       |            |         
f074e2ee33 |            |           |        | r:latest   |       |            |         
job-6a150e | altis-     | succeeded | Aug 30 | image://sh | <you> |            |         
a0-969e-4f | processor  |           |        | agren/alti |       |            |         
7a-b04d-e4 |            |           |        | s-processo |       |            |         
6cce9b593c |            |           |        | r:latest   |       |            |         

```

4. Header + horizontal and vertical lines
```
                   ID                    |      NAME       |  STATUS   |  WHEN  |                 IMAGE                  | OWNER | DESCRIPTION | COMMAND 
=========================================+=================+===========+========+========================================+=======+=============+=========
job-2a43301c-aaf1-4249-b2fc-86139200bc66 |                 | succeeded | Aug 16 | alpine:latest                          | <you> |             | sleep 2m
-----------------------------------------+-----------------+-----------+--------+----------------------------------------+-------+-------------+---------
job-45ff7ad4-ca48-41f8-8330-e22cb6288dad |                 | succeeded | Aug 28 | image://neuromation/fastai:latest      | <you> |             |         
-----------------------------------------+-----------------+-----------+--------+----------------------------------------+-------+-------------+---------
job-9e76a6cc-7550-4c81-8d3f-89dc6b59b6be |                 | succeeded | Aug 28 | image://neuromation/fastai:latest      | <you> |             |         
-----------------------------------------+-----------------+-----------+--------+----------------------------------------+-------+-------------+---------
job-8d49bc3c-b4f8-4067-89ec-fff074e2ee33 | altis-processor | succeeded | Aug 30 | image://shagren/altis-processor:latest | <you> |             |         
-----------------------------------------+-----------------+-----------+--------+----------------------------------------+-------+-------------+---------
job-6a150ea0-969e-4f7a-b04d-e46cce9b593c | altis-processor | succeeded | Aug 30 | image://shagren/altis-processor:latest | <you> |             |         

```
```
    ID     |    NAME    |  STATUS   |  WHEN  |   IMAGE    | OWNER | DESCRIPTIO | COMMAND 
           |            |           |        |            |       |     N      |         
===========+============+===========+========+============+=======+============+=========
job-2a4330 |            | succeeded | Aug 16 | alpine:lat | <you> |            | sleep 2m
1c-aaf1-42 |            |           |        | est        |       |            |         
49-b2fc-86 |            |           |        |            |       |            |         
139200bc66 |            |           |        |            |       |            |         
-----------+------------+-----------+--------+------------+-------+------------+---------
job-45ff7a |            | succeeded | Aug 28 | image://ne | <you> |            |         
d4-ca48-41 |            |           |        | uromation/ |       |            |         
f8-8330-e2 |            |           |        | fastai:lat |       |            |         
2cb6288dad |            |           |        | est        |       |            |         
-----------+------------+-----------+--------+------------+-------+------------+---------
job-9e76a6 |            | succeeded | Aug 28 | image://ne | <you> |            |         
cc-7550-4c |            |           |        | uromation/ |       |            |         
81-8d3f-89 |            |           |        | fastai:lat |       |            |         
dc6b59b6be |            |           |        | est        |       |            |         
-----------+------------+-----------+--------+------------+-------+------------+---------
job-8d49bc | altis-     | succeeded | Aug 30 | image://sh | <you> |            |         
3c-b4f8-40 | processor  |           |        | agren/alti |       |            |         
67-89ec-ff |            |           |        | s-processo |       |            |         
f074e2ee33 |            |           |        | r:latest   |       |            |         
-----------+------------+-----------+--------+------------+-------+------------+---------
job-6a150e | altis-     | succeeded | Aug 30 | image://sh | <you> |            |         
a0-969e-4f | processor  |           |        | agren/alti |       |            |         
7a-b04d-e4 |            |           |        | s-processo |       |            |         
6cce9b593c |            |           |        | r:latest   |       |            |         
```

**Comment by asvetlov** [comment]
Interesting. I think we should never wrap ID column: it is used by copy-paste often.
I think a version with lines looks better: otherwise IDs are not visually separated from each other.
If we never wrap IDs the version without lines may look be good too, let's see.

**Comment by shagren** [comment]
Like as it's fixed via #1099.
@ayushkovskiy can you reproduce?

**Comment by ayushkovskiy** [comment]
Confirm it will be fixed for the version > `19.9.23`


Version = `19.9.23`:
```
$ n ps
ID                                        NAME                                     STATUS   WHEN            IMAGE          OWNER  DESCRIPTION  COMMAND
job-2da71bab-2c63-43dd-94f0-8c44fd267410                                           running  50 seconds ago  ubuntu:latest  <you>               sleep 1h
job-1c2011cb-aa83-4923-aa87-18137a031137  test-test-test-test-test-test-test-test  running  26 seconds ago  ubuntu:latest  <you>               sleep 1h
```

Version `19.10.11a1` (on master branch, commit e5a16ab, upcoming release)
```
ID                                        NAME                  STATUS   WHEN            IMAGE          OWNER  CLUSTER  DESCRIPTION  COMMAND 
job-2da71bab-2c63-43dd-94f0-8c44fd267410                        running  27 seconds ago  ubuntu:latest  <you>  default               sleep 1h
job-1c2011cb-aa83-4923-aa87-18137a031137  test-test-test-test-  running  2 seconds ago   ubuntu:latest  <you>  default               sleep 1h
                                          test-test-test-test 
```

**Comment by ayushkovskiy** [comment]
@shagren,
> I played with textable. It can wrap cells if you have limited table width.

I think, the new output of `neuro ps` is pretty much fine and there's no problem to solve by introducing a new instrument (I mean `textable`). Let's do https://github.com/neuromation/platform-client-python/issues/1021 instead
